h/schemas/annotation.py

```diff
@@ -225,14 +225,6 @@ def _document(document, claimant):
     }
 
 
-def _format_jsonschema_error(error):
-    """Format a :py:class:`jsonschema.ValidationError` as a string."""
-    if error.path:
-        dotted_path = ".".join([str(c) for c in error.path])
-        return "{path}: {message}".format(path=dotted_path, message=error.message)
-    return error.message
-
-
```

Comment:
Found by the duplicate-code rule but this one was not used anywhere so it was an easy fix